trie/sync.go

```diff
@@ -552,11 +565,83 @@ func (s *Sync) commitCodeRequest(req *codeRequest) error {
 
 // ResolvePath resolves the provided composite node path by separating the
 // path in account trie if it's existent.
-func ResolvePath(path []byte) (common.Hash, []byte) {
+func ResolvePath(path []byte, isZk bool) (common.Hash, []byte) {
 	var owner common.Hash
+	if isZk {
+		if len(path) >= common.HashLength*8 {
+			owner = *zk.NewTreePath(path[:common.HashLength*8]).ToHash()
```

Comment:
owner seems to only be used if it's a PathScheme, is this necessary?
```suggestion
			
```

Reply:
You don't need to worry about pathschema. We need owner

Reply:
The owner means the owner of the storage tree. So, the first 32 bytes are the tree path of account tree.